File: pyjl/tests/test_performance/mandelbrot/simple_mandelbrot.py
MAX_ITER = 10000000

def mandelbrot(c):
    z = 0
    n = 0
    while abs(z) <= 2 and n < MAX_ITER:
        z = z*z + c
        n += 1
    return n

if __name__ == "__main__":
    map = []
    for a in range(-10, 10, 5):
        for b in range(-10, 10, 5):
            c = complex(a / 10, b / 10)
            map.append((c, mandelbrot(c)))

File: tests/performance_tests-py2many/simple_mandelbrot.jl
MAX_ITER = 10000000
function mandelbrot(c)::Int64
    z = 0
    n = 0
    while abs(z) <= 2 && n < MAX_ITER
        z = z * z + c
        n += 1
    end
    return n
end

function main()
    map = []
    for a in (-10:5:9)
        for b in (-10:5:9)
            c = complex(a / 10, b / 10)
            push!(map, (c, mandelbrot(c)))
        end
    end
end

main()
